content/blog/2024-12-03-tmil-11/index.md

```diff
@@ -0,0 +1,158 @@
+The documentation lays out more of the scope, goals, and main types provided by the crate.
+
+Since then, we have begun the work of making Peniko and Vello use the Color crate and improving it to support that usage.
+Early next year, we will use it within svgtypes, resvg, and tiny-skia.
+
+We will be releasing a 0.2.0 release within the next month and encourage people to give it a try.
+The 0.2.0 release will include some additional (non-CSS 4) color spaces like ACEScg, OkHsl, and OkHsv as well as things that we found useful when using it in Peniko and Vello.
+
+Many thanks to [Brendan Zabarauskas][] for working with us to transfer his `color` crate name for our use.
+His original crate was published as 0.0.1 on November 21, 2014, motivating us ten years later to ship our 0.1.0 on November 21, 2024.
+
+We're very excited about the future of this crate and look forward to talking to other people within the Rust ecosystem about using it.
+
+## Kurbo
+
+Kurbo has seen some minor updates within the last month and we're planning to publish them in a minor patch release within the next month.
+
+* [kurbo#390][] Reduce number of operations in `Triangle::circumscribed_circle`.
+* [kurbo#399][] Implement `Sum` for `Vec2`.
+
+Internally, we're talking about the future direction of Kurbo and how we can improve upon it as a foundational crate for curves, shapes, and math.
+Hopefully we'll be writing about that in more detail within the next couple of months.
+
+## Peniko
+
+Peniko will have a big release within the next month, once the new release of Color is out, that will remove the old `peniko::Color` type and replace it with the new create.
+This release will be the start of a path towards wider color gamut support throughout the Linebender ecosystem.
+
+We have long had a pain point with Peniko where it defines some resource management types, `Blob`, `Font`, and `WeakBlob`.
+These are used within Parley and Vello as well as by anyone using those crates.
+We plan to move these into a separate crate to ease versioning issues and to take advantage of this to improve the functionality.
+This will probably be coming in the next big release following the one adding Color.
+
+## Vello
+
+Vello too will have a big release within the next month, building upon the new releases of Peniko and Color.
+
+Additionally:
+
+* [vello#735][], [vello#743][]: Update to wgpu 23.0.1, matching what is used by Bevy 0.15.
+* [vello#722][]: Improving image sampling.
+* [vello#740][]: Correcting support for PNG glyphs in fonts, important for emoji on macOS.
+
+
+## Parley
+
+***SOMEONE ELSE WRITE THIS***
+
+## Masonry
+
+***SOMEONE ELSE WRITE THIS***
+
+## Xilem
+
+***SOMEONE ELSE WRITE THIS***
+
+## Piet
+
+While we aren't yet planning on a new release of Piet as there have been no changes since last month, we did begin a discussion about a possible major revision to Piet and what a next-generation Piet might look like.
```

Comment:
Talking about the absence of plans is even less useful than talking about plans. Let's cut it.

```suggestion
While there were no code changes, we did begin a discussion about a possible major revision to Piet and what a next-generation Piet might look like.
```